R/utils.R

```diff
@@ -551,7 +551,15 @@ verify_attr <- function(proposed, schema, layoutAttr = FALSE) {
     
     # do the same for "sub-attributes"
     if (identical(role, "object") && is.recursive(proposed[[attr]])) {
-      proposed[[attr]] <- verify_attr(proposed[[attr]], attrSchema, layoutAttr = layoutAttr)
+      # The "dimensions" attribute requires a special treatment as 
+      # it is an unnamed list and hence will be skipped by `for (attr in names(proposed))
+      if (attr == "dimensions") {
+        proposed[[attr]] <- lapply(proposed[[attr]], \(x) {
```

Comment:
`\(x)` was added fairly recently to R, so please use `function(x)` instead

Reply:
🙈 Thanks for the reminder. Should have thought of that myself.